### DISCUSSION TITLE
Seattle Public Libraries Web Application

### DISCUSSION THREAD

**Discussion by manhongg** [jakobzhao/geog495#55] [Final Project]
Link to repository: https://github.com/manhongg/geog495_final_project
Link to final project: Manhongg.github.io/geog495_final_project/index.html

<img width="1177" alt="web_application" src="https://user-images.githubusercontent.com/43800949/145517109-9fb8cab3-88a4-4cb9-adee-44f0c7bd3c10.png">

For my final project, I used a dataset detailing Seattle Public Library facilities to create a web application that displays the locations and addresses of all public libraries in the City of Seattle. I felt like this would be a useful web app to create because it provides insight into the accessibility of public libraries throughout Seattle's various neighborhoods. As someone who used to frequent public libraries and utilize their resources frequently, I thought it would be both interesting and insightful to create a web application that would allow users to better understand the geographic correlations between public libraries and community resources or the lack thereof between neighborhoods of varying socioeconomic conditions in a big city like Seattle.

**Comment by seanhy1** [comment]
Your design is intuitive in the sense for every library found on your web application provides data in both the sidebar and the popups. I did a similar project prior to the final project in regards to utilizing Seattle Libraries data, so seeing such data being optimized for this assignment is very intriguing to see! I also love that you added fly-to functionality for each library being triggered by click. Well done!

One change I would suggest, though it's not needed, is changing the color scheme of your side panel based on the customized marker points. matching a color of those books would be complementary to the eye, especially with the darker tone of base map you elected to go with. Again, it's not needed, but it would definitely improve the overall design of your application in my opinion!

**Comment by aembar** [comment]
Hi, I like icon chosen for public library locations and think the miles away field you have is really useful combined with the searchbar for pinging locations. Some things I would suggest are changing the base map to a street view to allow users to know the libraries neighborhoods better when traveling there. I also tried entering in some locations in Seattle and was not able to ping them, not sure if the bbox attribute for the geocoder is encompsing some areas.

**Comment by NahomAgidew** [comment]
Really smooth flow and great custom icons 📚 . An improvement would be: you see how when you zoom out the icons are very large? It would be better if you reduced the sizes.
![image](https://user-images.githubusercontent.com/11695305/145707823-24d8f343-efbb-4a29-9117-ad66d55723a4.png)

**Comment by whalen323** [comment]
As someone who works in a library, I love this! It's really easy to use and understand, and the adding of a search bar tool really helps in looking for a specific location! The only real comment that I have is about the icons- they are cool looking but seem a bit large relative to the map size, so it might look better to reduce them a bit.

**Comment by VkSath00** [comment]
I like the way your map looks as its simple enough to follow and use, having the ability to search makes it easy to find places as well. The search bar that you've included is put together and very well done. I think the icons could be a little smaller when you're zooming out of your map. Otherwise its very well done
